parachain/pallets/omni-account/src/lib.rs

```diff
@@ -0,0 +1,216 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
+pub use core_primitives::{Identity, MemberIdentity};
+pub use frame_system::pallet_prelude::BlockNumberFor;
+pub use pallet::*;
+
+use frame_support::pallet_prelude::*;
+use frame_system::pallet_prelude::*;
+use sp_core::H256;
+use sp_core_hashing::blake2_256;
+use sp_std::vec::Vec;
+
+pub trait AccountIdConverter<T: Config> {
+	fn convert(account_id: T::AccountId) -> Identity;
+}
+
+#[frame_support::pallet]
+pub mod pallet {
+	use super::*;
+
+	/// The current storage version.
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
+
+	#[pallet::pallet]
+	#[pallet::storage_version(STORAGE_VERSION)]
+	#[pallet::without_storage_info]
+	pub struct Pallet<T>(_);
+
+	#[pallet::config]
+	pub trait Config: frame_system::Config {
+		/// The event type of this pallet.
+		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
+		/// The origin which can manage the pallet.
+		type TEECallOrigin: EnsureOrigin<Self::RuntimeOrigin>;
+		/// The maximum number of identities an id graph can have.
+		#[pallet::constant]
+		type MaxIDGraphLength: Get<u32>;
+		/// AccountId converter
+		type AccountIdConverter: AccountIdConverter<Self>;
+	}
+	pub type IDGraphLinks<T> = BoundedVec<(H256, MemberIdentity), <T as Config>::MaxIDGraphLength>;
```

Comment:
Shall we use a struct to replace `(H256, MemberIdentity)`? Like
```
struct MemberAccount {
  id: MemberIdentity,
  hash: H256,
}
```

Maybe better for future expansion, but it's a flavor problem